resources/benchmark.js

```diff
@@ -0,0 +1,148 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const { Suite } = require('benchmark');
+const beautifyBenchmark = require('beautify-benchmark');
+const { execSync } = require('child_process');
+const os = require('os');
+const path = require('path');
+
+// Like build:cjs, but includes __tests__ and copies other files.
+const BUILD_CMD = 'babel src --optional runtime --copy-files --out-dir dist/';
+const LOCAL = 'local';
+const LOCAL_DIR = path.join(__dirname, '../');
+const TEMP_DIR = os.tmpdir();
```

Comment:
@leebyron Both #1167 and #1163 were intended to enable benchmarking during development and it really inconvenient if after every small change you need to rebuild `HEAD` 😞